client/ayon_houdini/api/lib.py

```diff
@@ -146,11 +146,26 @@ def validate_fps():
     return True
 
 
-def render_rop(ropnode):
+def render_rop(ropnode, frame_range=None):
```

Comment:
Why don't we just match the defaults of houdini?
```suggestion
def render_rop(ropnode, frame_range=tuple()):
```

Reply:
yeah, correct. I forgot that tuples are immutable so this won't make us fall in the pass by reference trap. 
Many thanks... https://github.com/ynput/ayon-houdini/pull/45/commits/61c2fae3f61405c70b31f814ea8931e3f71ce34e